### ISSUE TITLE
Objects exported from goobi have not started accessioning

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/dor-services-app#3100]
@calavano has done some initial troubleshooting, and thinks what is happening is on our end. He notes that Goobi puts the files in /dor/assembly and creates the stubContentMetadata.xml file, and makes the POST request to dor-services-prod, and that there are no error messages from Goobi for any of these steps. However, the items remain stuck in this step and do not complete accessioning. It appears that we have not been able to successfully accession anything from Goobi since 12/3/2021. 

Original list: 

```
ds702dt7257
ft748nh4404
gg698jc5133
jg787pr8897
mf235hy1884
mk292tp3606
pp603ts4260
qf475kk3918
qg935xb4586
zs226gk6973
zv186tj9590
```

No actions taken on  yet:

```
jg787pr8897
mf235hy1884
mk292tp3606
pp603ts4260
qf475kk3918
qg935xb4586
zs226gk6973
zv186tj9590
```

these originally got stuck, but have now been manually unstuck by starting accessioning (see comments below): 

```
ft748nh4404
gg698jc5133
```

Note: this one did not have valid stubContentMetadata: `ds702dt7257` but when i removed the folder in dor/assembly and tried to export again from goobi, it failed in goobi.  we then started accession on it, and it failed due to lack of stub content metadata

**Comment by peetucket** [comment]
For `ft748nh4404`:

1. found content and stub content metadata in dor/assembly workspace
2. did not find any evidence of dor-services-app being called with the start accession method
3. tried to make this call manually from argo:
```
resp = Dor::Services::Client.object('druid:ft748nh4404').accession.start
```
4. Noted that this correctly started assemblyWF for the object: https://argo.stanford.edu/view/druid:ft748nh4404

So not clear why goobi export created the content on the file system but did not correctly make the call to dor-services-app

**Comment by dinahhandel** [comment]
@peetucket given what you say above, should we ticket this in the [sul-goobi issues queue](https://github.com/sul-dlss/sul-goobi/issues)?

**Comment by peetucket** [comment]
Possibly.  We are still trying to do some more troubleshooting.  I also noted that the first item on the list `ds702dt7257` did not have any stubContentMetadata in the dor/assembly folder, and when i tried to click the export button in goobi, it spun and spun and eventually timed out and gave an error (this was after clearing out that druid's folder in dor/assembly to start from a clean slate).

**Comment by peetucket** [comment]
From the goobi server, we were able to confirm that dor-services-prod is accessible (i.e. no obvious networking issue):

```
curl https://dor-services-prod.stanford.edu/status/all
Default Collection
  background_jobs: PASSED Sidekiq queue 'default' latency at reasonable level (0) (0.013s)
  database: PASSED Schema version: 20200521153735 (0.023s)
  default: PASSED Application is running (0.000s)
  external-symphony: PASSED HTTP check successful (0.022s) (OPTIONAL)
  feature-tables-have-data: PASSED BackgroundJobResult has data. TagLabel has data. Event has data. AdministrativeTag has data. (0.020s)
  rabbit: PASSED Rabbit Connection Status: (open) (0.023s)
```

We then used the dor-services-prod token that goobi has configured in `/opt/digiverso/goobi/config/plugin_intranda_export_stanford.xml` to make a accession call to dor-services-prod and verify that this worked ok too.

**Comment by peetucket** [comment]
Bottom line: we do not think there is anything amiss on our end, but goobi should look at their logs to see what errors may be happening one export:

1. Why did `ds702dt7257` not produce any stub contentmetadata and why did it fail again on export?
2. Is the call to dor-services-prod working ok?  We verified the token is still valid and there do not appear to be any issues with networking preventing the server from being reached, but perhaps something else is happening when they make the call?  We did note that when you make the call via CURL, it looks like this:

```
curl -X POST -H "Authorization: YOUR_TOKEN_HERE" -H "Content-Type: application/xml" https://dor-services-prod.stanford.edu/v1/objects/druid:DRUID_HERE/accession
```

Maybe they can verify the authorization header is of the correct format (this may have changed at some point on our end)?

**Comment by peetucket** [comment]
Note, we should be able to unblock druids in the list above by manually starting accession on them (assuming they have valid output including stubContentMetadata in the /dor/assembly folder).  But we will await to see if you want us to do this in case we need those examples for troubleshooting.

**Comment by peetucket** [comment]
We also tried to re-export this object: jg787pr8897 from goobi and noted that we got a 500 error from goobi (in the UI) after a long wait.  We also noticed that only a single file with a new timestamp was actually exported to the /dor/assembly filesystem for that object...so something happened, but the export from goobi seems to fail part way and before the call to start accessioning is occurring.  We aren't sure why.
